tensornetwork/linalg/initialization.py

```diff
@@ -17,7 +17,7 @@
 from typing import Optional, Sequence, Tuple, Any, Union, Type, Callable, List
 from typing import Text
 import numpy as np
-from tensorflow.python.ops.numpy_ops.np_arrays import convert_to_tensor
```

Comment:
why is there a global `tensorflow` import? The only place where `tensorflow`, `jax` and `pytorch` should be imported is inside the respective backends.